### ISSUE TITLE
XHR demo success text not announced by screen reader

### ISSUE THREAD

**Issue by dmochak** [ampproject/amphtml#3873]
If a screen-reader user completes the XHR demo, the text that confirms this is not read automatically.

https://amp-mk-1.herokuapp.com/examples.build/forms.amp.max.html

1. Fill out the XHR demo (bottom of page) correctly with a screen reader active.
2. select subscribe.
3. The page is updated with the success text, however the screenreader does not read it. You must manually navigate to the end of the page in order to discover whether you succeeded or not. 

repros in Firefox/Chrome, likely effects all browsers.
Powered by AMP ⚡ HTML – Version 1467182714735

**Comment by A11yravi** [comment]
Hello Team,

Same issue appear when tested on Android tablet 
OS : Android 6.0
Device: Nexus 9 (tablet)
Browser: Chrome (Version 51.0.2704.81)

but, worked fine when tested on Android phone
Device: Nexus 5x
OS: Android 6.0.1
Build Number MHC19T

Please let me know for any more information needed.

Thanks,
Ravi
